tests/system_test.py

```diff
@@ -1,3 +1,3 @@
-def test_status_endpoint(client):
+def test_status_endpoint(client, db_session):
```

Comment:
What does this change do?

Reply:
it opens the connection to the test DB before the request is made, and closes it after - this test was failing with "DB connection is closed" after I added the new DB fixtures